lib/native_images.dart

```diff
@@ -1,68 +0,0 @@
-import 'dart:async';
-import 'dart:typed_data';
-import 'dart:ui' as ui;
-
-import 'package:flutter/rendering.dart';
-import 'package:flutter/services.dart';
-import 'package:flutter/painting.dart';
-import 'package:flutter/foundation.dart'
-    show SynchronousFuture, describeIdentity;
-
```

Comment:
All public classes/methods in this file need dartdocs

Reply:
Added docstrings, tried to add you as reviewer for the PR.  You need to accept being a collaborator.